server/index.js

```diff
@@ -19,8 +19,14 @@ passport.use(
       consumerSecret: `${process.env.TWITTER_API_SECRET_KEY}`,
       callback: `${process.env.BASE_SERVER_URL}/oauth/twitter/callback`,
     },
-    function (token, tokenSecret, profile, cb) {
-      return cb(null, profile);
+    async function (token, tokenSecret, profile, cb) {

```

Comment:
nit: use an async arrow function

Reply:
Done, and converted a few others as well.